R/reportEmi.R

```diff
@@ -1001,65 +996,54 @@ reportEmi <- function(gdx, output = NULL, regionSubsetList = NULL, t = c(seq(200
 
 
 
-  # sinks/usage of captured carbon
+  # storage/usage of captured carbon
   out <- mbind(out,
                # underground carbon storage
                setNames(dimSums(vm_co2CCS, dim = 3, na.rm = T) * GtC_2_MtCO2,
-                          "Carbon Management|Carbon Sinks|+|Storage (Mt CO2/yr)"),
+                          "Carbon Management|Storage (Mt CO2/yr)"),
                # carbon in synfuels
                setNames(collapseDim(dimSums(vm_co2CCUshort[, , c("MeOH", "h22ch4")], dim = 3, na.rm = T)) * GtC_2_MtCO2,
-                          "Carbon Management|Carbon Sinks|+|CCU (Mt CO2/yr)"),
+                          "Carbon Management|Usage (Mt CO2/yr)"),
                # carbon in synthetic liquids
                setNames(collapseDim(vm_co2CCUshort[, , "MeOH"]) * GtC_2_MtCO2,
-                          "Carbon Management|Carbon Sinks|CCU|+|Synthetic Liquids (Mt CO2/yr)"),
+                          "Carbon Management|Usage|+|Synthetic Liquids (Mt CO2/yr)"),
                # carbon in synthetic liquids
                setNames(collapseDim(vm_co2CCUshort[, , "h22ch4"]) * GtC_2_MtCO2,
-                          "Carbon Management|Carbon Sinks|CCU|+|Synthetic Gases (Mt CO2/yr)"),
-               # total CO2 used
-               setNames((dimSums(vm_co2CCS, dim = 3, na.rm = T) + dimSums(vm_co2CCUshort, dim = 3, na.rm = T)) * GtC_2_MtCO2,
-                          "Carbon Management|Carbon Sinks (Mt CO2/yr)")
+                          "Carbon Management|Usage|+|Synthetic Gases (Mt CO2/yr)")
   )
 
-  # total carbon captured and CCS share
+  # share of stored carbon from total captured carbon
   out <- mbind(out,
 
-               # total captured CO2
-               setNames(dimSums(vm_co2capture, dim = 3) * GtC_2_MtCO2,
-                          "Carbon Management|Carbon Capture (Mt CO2/yr)"),
-               # share of stored carbon from total captured carbon
                setNames(p_share_CCS * 100,
                           "Carbon Management|Share of Stored CO2 from Captured CO2 (%)")
-
   )
 
   # calculate some specific carbon sink (CCS) variables
   out <- mbind(out,
 
                # total captured and stored by pe2se BECCS
-               setNames(out[, , "Carbon Management|Carbon Sources|+|Biomass|Pe2Se (Mt CO2/yr)"] * p_share_CCS,
-                          "Carbon Management|Carbon Sinks|Storage|+|Biomass|Pe2Se (Mt CO2/yr)"),
-               setNames(out[, , "Carbon Management|Carbon Sources|+|Fossil|Pe2Se (Mt CO2/yr)"] * p_share_CCS,
-                          "Carbon Management|Carbon Sinks|Storage|+|Fossil|Pe2Se (Mt CO2/yr)"),
-               setNames(out[, , "Carbon Management|Carbon Sources|+|Industry Energy (Mt CO2/yr)"] * p_share_CCS,
-                          "Carbon Management|Carbon Sinks|Storage|+|Industry Energy (Mt CO2/yr)"),
+               setNames(out[, , "Carbon Management|Carbon Capture|+|Biomass|Pe2Se (Mt CO2/yr)"] * p_share_CCS,
```

Comment:
I think it would make sense to call the storage and usage variables "Carbon Management|Carbon Capture|Storage" and "Carbon Management|Carbon Capture|Usage". This would make it clear that "Carbon Management|Carbon Capture" = "Carbon Management|Carbon Capture|Storage" + "Carbon Management|Carbon Capture|Usage"